src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -578,20 +581,28 @@ public async Task<ListResponseModel<OrganizationUserBulkResponseModel>> BulkRemo
     [RequireFeature(FeatureFlagKeys.AccountDeprovisioning)]
     [HttpDelete("{id}/delete-account")]
     [HttpPost("{id}/delete-account")]
-    public async Task DeleteAccount(Guid orgId, Guid id)
+    public async Task<IActionResult> DeleteAccount(Guid orgId, Guid id)
```

Comment:
@eliykat @jrmccannon 

Please review the app code only, not the test code. Start with the controller endpoints for single and multiple removals, then work your way down to the command and validation layers.

The main goal of this first review is to catch any issues early. If everything looks good, I’ll proceed with updating the tests. As you can see, this is a significant refactor.